packages/now-build-utils/src/file-ref.ts

```diff
@@ -27,24 +28,34 @@ export default class FileRef implements File {
   public mode: number;
   public digest: string;
   public mutable: boolean;
+  public ephemeral: boolean;
 
-  constructor({ mode = 0o100644, digest, mutable = false }: FileRefOptions) {
+  constructor({
+    mode = 0o100644,
+    digest,
+    mutable = false,
+    ephemeral = false,
+  }: FileRefOptions) {
     assert(typeof mode === 'number');
     assert(typeof digest === 'string');
     assert(typeof mutable === 'boolean');
+    assert(typeof ephemeral === 'boolean');
     this.type = 'FileRef';
     this.mode = mode;
     this.digest = digest;
     this.mutable = mutable;
+    this.ephemeral = ephemeral;
   }
 
   async toStreamAsync(): Promise<NodeJS.ReadableStream> {
     let url = '';
     // sha:24be087eef9fac01d61b30a725c1a10d7b45a256
     const digestParts = this.digest.split(':');
```

Comment:
While you're changing this code, can you change this line?

```suggestion
    const [digestType, digestHash] = this.digest.split(':');
```